### PR TITLE
I18n release

### PR DESCRIPTION
**Pull request by danimoh**

### CODE REVIEW

**Review by sisou — APPROVED** [review]

**Comment by danimoh** [comment]
I repurposed this PR to a general PR for "I18n release" changes similar to the one in the Hub.
@sisou please re-review the newest changes.

Additional changes might be added later depending on feedback

File: src/translations/de.json [hunk 1]
```diff
@@ -22,21 +22,23 @@
     "create-wallet-desc": "Das ist dein neues Konto mit deiner ersten Adresse.",
 
     "import-heading-enter-recovery-words": "Wiederherstellungswörter eingeben",
-    "import-import-login-file": "Lade dein Login-File hoch",
+    "import-import-login-file": "Importiere mittels Login‑Datei",
```
Comment (sisou) [inline src/translations/de.json]: I think the wording should be about "login", not "import". So rather "Log dich mittels Login-Datei ein"

**Review by sisou — APPROVED** [review]

**Review by mraveux — APPROVED** [review]

**Comment by danimoh** [comment]
This should now be all changes for the release.
Please re-review.

File: src/translations/fr.json [hunk 2]
```diff
@@ -3,7 +3,15 @@
 
     "language-changed": "La langue d'affichage a changé. Voulez-vous recharger la page pour mettre à jours toutes les traductions ? Sinon certaines pourraient ne pas être mises à jour automatiquement.",
 
-    "back-to": "Retour à ",
+    "back-to-app": "Back to {appName}",
```
Comment (mraveux) [inline src/translations/fr.json]: This one and the others below are now translated in transifex.

File: src/translations/fr.json [hunk 3]
```diff
@@ -158,8 +166,7 @@
     "remove-key-download-login-file": "Télécharger",
     "remove-key-recovery-words-question": "Savez-vous où sont vos Mots de Récupération ?",
     "remove-key-show-recovery-words": "Créer une sauvegarde",
-    "remove-key-enter-label-1": "Écrivez",
-    "remove-key-enter-label-2": "pour vous déconnecter.",
+    "remove-key-label-confirm-instructions": "Type \"{accountLabel}\" to log out.",
```
Comment (mraveux) [inline src/translations/fr.json]: Translated too

**Review by mraveux — APPROVED** [review]
Just some missing French translations, otherwise it's all good.

File: src/request/TopLevelApi.js [hunk 4]
```diff
@@ -229,15 +229,60 @@ class TopLevelApi extends RequestParser {
...
+    enableGlobalCloseButton(requestOrCustomButtonText) {
         /** @type {HTMLElement} */
         const $globalCloseText = (document.querySelector('#global-close-text'));
         /** @type {HTMLSpanElement} */
         const $button = ($globalCloseText.parentNode);
         if (!$button.classList.contains('display-none')) return;
-        $globalCloseText.textContent = buttonText;
+
+        // eslint-disable-next-line require-jsdoc-except/require-jsdoc
+        const setButtonText = () => {
```
Comment (sisou) [inline src/request/TopLevelApi.js:242]:
```suggestion
        function setButtonText() {
```

File: src/request/remove-key/RemoveKey.js [hunk 5]
```diff
@@ -58,7 +59,18 @@ class RemoveKey {
             $loginFileContainer.classList.add(colorClass);
         }
 
-        $labelSpan.textContent = this._request.keyLabel;
+        // eslint-disable-next-line require-jsdoc-except/require-jsdoc
+        const setLabelConfirmInstructions = () => {
```
Comment (sisou) [inline src/request/remove-key/RemoveKey.js:63]:
```suggestion
        function setLabelConfirmInstructions() {
```

File: src/translations/de.json [hunk 6]
```diff
@@ -3,15 +3,23 @@
 
     "language-changed": "Die Anzeigesprache hat sich geändert. Möchtest du die Seite neu laden um alle Übersetzungen zu aktualisieren? Andernfalls könnten manche Übersetzungen nicht automatisch aktualisiert werden.",
 
-    "back-to": "Zurück zu",
+    "back-to-app": "Zurück zu {appName}",
+    "back-to-accounts": "Zurück zu meinen Konten",
+    "back-to-wallet": "Zurück zur {Wallet}",
+    "back-to-miner": "Zurück zum {Nimiq Miner}",
+    "back-to-faucet": "Zurück zur {Nimiq Faucet}",
+    "back-to-donation": "Zurück zur Spende",
```
Comment (sisou) [inline src/translations/de.json]: @danimoh I believe this is not a good translation. This is the _Donation Button Maker app_, not a donation itself.

File: src/translations/de.json [hunk 7]
```diff
@@ -3,15 +3,23 @@
...
+    "back-to-gift-card": "Zurück zur Geschenkkarte",
+    "back-to-vote": "Zurück zur Abstimmung",
+    "back-to-cpl": "Zurück zu CryptoPayment.link",
```
Comment (sisou) [inline src/translations/de.json:14]: @danimoh I think this is too long, this would be shorter:
```suggestion
    "back-to-cpl": "Zurück zum Link",
```

**Review by sisou — APPROVED** [review]

File: src/request/TopLevelApi.js (hunk 4 above)
Comment (danimoh) [reply src/request/TopLevelApi.js:242]: In my opinion the way it is currently is the nicer way of defining a function within a function.

File: src/translations/de.json (hunk 6 above)
Comment (danimoh) [reply src/translations/de.json]: The app itself is called `Nimiq Donation`, thus this is a translation close to the original `Back to Nimiq Donation`.
However, to resolve this, I the donation app now identifies itself as `Donation Button Creator` and I adapted the translations accordingly.

File: src/translations/de.json (hunk 7 above)
Comment (danimoh) [reply src/translations/de.json:14]: That's too unspecific imo and `Zurück zu CryptoPayment.link` still fits on mobile.

**Pull request merged**